.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+
+go:
+  - 1.x
+  - master
+
+install: go get -t ./...
```

Comment:
is this even necessary?

Reply:
Yes, if I don't add it, I get the following error:

```
go test -cover -covermode=count -coverprofile=coverage.out .
detection.go:23:2: cannot find package "github.com/golang/protobuf/proto" in any of:
	/home/travis/.gimme/versions/go1.9.2.linux.amd64/src/github.com/golang/protobuf/proto (from $GOROOT)
	/home/travis/gopath/src/github.com/golang/protobuf/proto (from $GOPATH)
make: *** [test] Error 1
The command "make" exited with 2.
```